transport/x/grpc/handler.go

```diff
@@ -116,28 +126,66 @@ func (h *handler) getTransportRequest(ctx context.Context, decodeFunc func(inter
 	return transportRequest, nil
 }
 
-func (h *handler) call(ctx context.Context, transportRequest *transport.Request) (interface{}, error) {
+func (h *handler) call(ctx context.Context, transportRequest *transport.Request, responseMD metadata.MD) (interface{}, error) {
 	handlerSpec, err := h.router.Choose(ctx, transportRequest)
 	if err != nil {
 		return nil, err
 	}
 	switch handlerSpec.Type() {
 	case transport.Unary:
-		return h.callUnary(ctx, transportRequest, handlerSpec.Unary())
+		return h.callUnary(ctx, transportRequest, handlerSpec.Unary(), responseMD)
 	default:
-		return nil, errors.UnsupportedTypeError{"grpc", handlerSpec.Type().String()}
+		return nil, yarpcerrors.UnimplementedErrorf("transport:grpc type:%s", handlerSpec.Type().String())
```

Comment:
Will this error be transparent to the most casual user calling this service? I would expect something to see something like `service "kitchen-sink" over transport protocol "grpc" does not implement "oneway" handlers`